src/jira/client/axios.ts

```diff
@@ -147,13 +147,16 @@ const instrumentRequest = (response) => {
 const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 	return async (error: AxiosError) => {
 		instrumentRequest(error?.response);
-		if (error.response?.status === 503) {
+		if (error.response?.status === 503 || error.response?.status === 405) {
```

Comment:
Ideally this should be feature-flagged...

Reply:
Why?

Reply:
Because we are changing the behaviour?

Reply:
I'm considering it fine because of the comment below.